.gitignore

```diff
@@ -1,3 +1,4 @@
 _site/
 .sass-cache/
 .jekyll-metadata
+*~
```

Comment:
This might, once again, make more sense as a completely separate PR, but probably less important in this case since it's paired with the style guide.